Fabric.Authorization.AccessControl/src/app/modules/access-control/custom-group/custom-group.component.spec.ts

```diff
@@ -263,6 +263,36 @@ describe('CustomGroupComponent', () => {
         + `${mockUsersResponse[0].identityProviderUserPrincipalName}`);
     });
 
+    it('dont reset the roles if group name invalid', () => {
```

Comment:
YOU DID IT!  Great job!  Looks like we werent too far off.  Thats awesome!

Reply:
Thanks to you and your help, I appreciate it.